bigchaindb/common/transaction.py

```diff
@@ -508,20 +503,20 @@ def validate_asset(self, amount=None):
                                   ' greater than one')
 
 
-class AssetLink(Asset):
+class AssetLink(object):
```

Comment:
I'm thinking it could be a good idea to bust out some `namedtuple`s :smiling_imp: